core/src/test/java/org/frankframework/receivers/ReceiverTest.java

```diff
@@ -606,6 +620,69 @@ public void run() {
 		verify(jmsMessage, atLeastOnce()).acknowledge();
 	}
 
+	@Test
```

Comment:
```suggestion
	@Test
	// also ensures that we use our own EventErrorHandler.
```

Maar dan moet je ook de `testApplicationContext.xml` aanpassen :P

Reply:
Als we de `testApplicationContext.xml` aanpassen dan passen we dat aan voor alle tests, nu alleen maar de 1 test waar we de error-handler nodig hebben. En definieren van die bean in de context geeft hem misschien ook automatisch een task-executor die uitvoering async maakt, en dat maakt de tests weer lastiger.

Dit leek mij de meest eenvoudige oplossing voor de test.